license_sh/helpers.py

```diff
@@ -92,7 +95,10 @@ def annotate_dep_tree(tree, whitelist: [str]) -> Tuple[AnyNode, Set[str]]:
 
     licenses_not_found = set()
     for node in PreOrderIter(tree):
-        node.license_problem = not is_license_ok(node.license_normalized, whitelist)
+        node.license_problem = (
+            not is_license_ok(node.license_normalized, whitelist)
+            and node.name not in ignored_packages
```

Comment:
Node.name concist of name and version? If not this is wrong

Reply:
@S4n60w3n as I wrote here https://github.com/webscopeio/license.sh/issues/41#issuecomment-566527158, we will support only names in the config for now. 

Each language has different versioning mechanism, so I'd start with a name and continue with supporting semver later.